### ISSUE TITLE
Ne pas prendre en compte les liens

### ISSUE THREAD

**Issue by mildis** [coti/digitalbot#7]
Exclure les URL de l’analyse afin de bien identifier textuellement et immédiatement la source du déclenchement.
cf piece jointe : il faut ouvrir le lien pour comprendre que c’est parce qu’il contient digital.
![capture d ecran 2017-04-30 a 15 59 01](https://cloud.githubusercontent.com/assets/1887285/25564930/0643db7e-2dbe-11e7-86da-3ffc175fd6df.png)

**Comment by coti** [comment]
oui, il faut que je bricole un bout de regex pour exclure quand c'est dans un lien...

**Comment by mildis** [comment]
Sinon je me demandais comment réagissait l’API de recherche si le bot cherchait ‘ digital’ (avec une espace avant). Ça pourrait faire un premier filtre, notamment pour les handles et le lien si l’espace est bien prise en compte.

**Comment by coti** [comment]
Bonne question, je vais jeter un oeil... Sinon j'ai bricolé ça
https://github.com/coti/digitalbot/commit/89cdb26e02ce654c9a4bb1a879bc8b67660916c1

**Comment by mildis** [comment]
Question subsidiaire : c’est bien l’url de destination qui est fournie par l’API ?
Si c’est l’url raccourcie via bit.ly ou autre, c’est inutile.

**Comment by coti** [comment]
non apparemment c'est complet
